src/content/deployment/flavors-ios.md

```diff
@@ -0,0 +1,471 @@
+      **Runner**.
+
+    * Open the **General** tab if it's not already open.
+
+    * Go to the **Apps Icons and Launch Screen** section and
+      expand it.
+
+    * To the right of the **App icon** field, click
+      **+** and update the fields as follows:
+
+      * **Debug-staging**: `AppIcon-staging`
+      * **Profile-staging**: `AppIcon-staging`
+      * **Release-staging**: `AppIcon-staging`
+      * **Debug-production**: `AppIcon-production`
+      * **Profile-production**: `AppIcon-production`
+      * **Release-production**: `AppIcon-production`
+
+1.  Launch the app for each scheme (`staging`, `production`)
+    and check to make sure that the app display name has
+    changed for each. To launch a scheme, see the steps in
+    [Launch an Xcode scheme][].
+
+[Launch an Xcode scheme]: #launch-an-xcode-scheme
+[App Icon Generator]: https://www.appicon.co/
+
+### Add unique build settings
+
+You can use [build settings][] to govern your iOS build
+process from compilation and linking to debugging and
+distribution. One way that you can use build settings
+with Flutter flavors is to assign those build settings
+to Xcode build configurations. For example, you might want
+to assign different API URLs to  `Debug-staging` and
+`Debug-production`. For example: 
+
+```console title="debug-staging-settings.xcconfig"
+# Debug-staging build settings
+API_BASE_URL = staging.flavors.com/api
+```
+
+```console title="debug-production-settings.xcconfig"
+# Debug-production build settings
+API_BASE_URL = flavors.com/api
+```
+
+If you would like to add additional build settings for
+a specific build configuration, see Apple's
+[Adding a build configuration file to your project][].
+
+[build settings]: https://developer.apple.com/documentation/xcode/build-settings-reference/
+[Adding a build configuration file to your project]: https://developer.apple.com/documentation/xcode/adding-a-build-configuration-file-to-your-project
+
+### Add additional customizations
+
+This document contains a few common Xcode scheme
+configurations, but there are many more that you can apply.
+To learn about them, see
+[Customizing the build schemes for a project][].
+
+[Customizing the build schemes for a project]: https://developer.apple.com/documentation/xcode/customizing-the-build-schemes-for-a-project
+
+### Update Podfiles
+
+If you are creating new Xcode schemes for a Flutter iOS
+project and you have an iOS Podfile in an existing
+Flutter project, you must update the Flutter iOS Podfile to
+match the changes you made in Xcode.
+
+The following steps show how to update your iOS Podfile to
+include two new Xcode schemes called `staging` and
+`production` in a Flutter project called `flavors_example`.
+You can also use these steps to update a macOS
+project by replacing any reference to `iOS` with `macOS`.
+
+1. In your IDE, open the `ios/Podfile` file.
+1. Make the following updates and save your changes.
+
+    ```json title="flavors_example/ios/Podfile"
```

Comment:
I believe Podfiles are some sort of Ruby DSL:

````suggestion
    ```ruby title="flavors_example/ios/Podfile"
````

Reply:
Fixed! And finally, the highlighting looks good for this one.